algorithms/src/std_algorithms/impl/Kokkos_ValueWrapperForNoNeutralElement.hpp

```diff
@@ -29,12 +29,6 @@ template <class Scalar>
 struct ValueWrapperForNoNeutralElement {
   Scalar val;
   bool is_initial = true;
-
-  KOKKOS_FUNCTION
-  void operator=(const ValueWrapperForNoNeutralElement& rhs) {
-    val        = rhs.val;
-    is_initial = rhs.is_initial;
-  }
```

Comment:
Why is removing the right thing to do?
Not used?  Unnecessary?  Please elaborate.

Reply:
Again, the compiler-generated special member functions are sufficient. The alternative for fixing the warning would be to define a copy constructor.